src/Config/Config.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PharIo\ComposerDistributor\Config;
+
+use PharIo\ComposerDistributor\FileList;
+
+class Config
+{
+	/** @var string */
+	private $package;
+
+	/** @var string */
+	private $keyDirectory;
+
+	/** @var \PharIo\ComposerDistributor\FileList */
+	private $phars;
+
+	public function __construct(string $package, string $keyDir, FileList $phars)
+	{
+		$this->package      = $package;
+		$this->keyDirectory = $keyDir;
```

Comment:
And here perhaps at least a check whether the folder or file actually exists?

Reply:
Done